### ISSUE TITLE
page "number" selector (EPUB authored page list):  custom select/option instead of native, space doesn't select from list (types character), and enter selects but also submits (premature)

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#2342]

**Comment by arthur-lemeur** [comment]
About the "enter" which submits automatically the change of page, this is a hardcoded behaviour : 
`        <form
            id="gotoPageForm"
            onSubmit={(e) => {
                e.preventDefault();
            }
            }
            onKeyUp=
                {
                    (e) => {
                        // SPACE does not work (only without key mods on button)
                        //  || e.key === "Space"
                        if (e.key === "Enter") {
                            const closeNavGotoPage = !dockedMode && !(e.shiftKey && e.altKey);
                            e.preventDefault();
                            handleSubmitPage(closeNavGotoPage);
                        }
                }
            }`

Do you want to delete that ?

**Comment by danielweck** [comment]
Well, in Thorium2 the page list selector was a native control which handled SPACE key strokes (and ARROWs) so the ENTER key event handler was added to submit the form without having to move keyboard focus to the "goto" button.
Now with custom controls we need to rethink this pattern.

**Comment by llemeurfr** [comment]
Hi, "go to page" is not a pure Select box, but an Autocomplete (datalist) box. If I refer to https://developer.mozilla.org/en-US/docs/Learn/Forms/Other_form_controls#autocomplete_box, a space key in the input field adds a space to the input, while using the space key in the list of proposed options has no effect (return selects the option). 

My conclusion is that the current behavior follows the expected UX (even if the HTML generated is not a datalist, thanks to React...).

**Comment by danielweck** [comment]
Yes, the problem is that the ENTER key is bound to form submission so we can't even select an option from the list anymore :)
